src/plots/cartesian/layout_attributes.js

```diff
@@ -180,7 +180,7 @@ module.exports = {
         description: [
             'Sets the text displayed at the ticks position via `tickvals`.',
             'Only has an effect if `tickmode` is set to *array*.',
```

Comment:
actually while editing this, could I rm 'Only has an effect if `tickmode` is set to *array*.' cause that's actually not true anymore right @etpinard // @mdtusz